src/shogun/multiclass/CoverTreeKNNSolver.h

```diff
@@ -0,0 +1,55 @@
+/* This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Copyright (c) 2012-2013 Sergey Lisitsyn
+ */
+
+
+#ifndef COVERTREESOLVER_H__
+#define COVERTREESOLVER_H__
+
+#include <shogun/lib/config.h>
+
+#include <shogun/lib/common.h>
+#include <shogun/distance/Distance.h>
+#include <shogun/multiclass/KNNSolver.h>
+
+namespace shogun
+{
+
+/**
+ * The CoverTreeKNNSolver class is a solver class which inherit from KNNSolver class. 
+ * It use the cover tree to speed up the nearest neighbor search and you can find more
```

Comment:
It uses cover trees to speed up the nearest neighbour search (typo and grammar mistake).
Make it two sentences: For more information, see [link]